### PR TITLE
added index html page

### PR DESCRIPTION
**Pull request by dmison**
As per issue: https://github.com/jboss-jdf/jboss-as-quickstart/issues/288

Signed-off-by: Darrin Mison dmison@me.com

### CODE REVIEW

File: logging-tools/src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+         courtesy of <a href="http://translate.google.com">http://translate.google.com</a> for demonstration.
+         My apologies if they are less than ideal translations.
+         </p>
+         
+         <p>
+         The different methods of the JAX-RS service can be access using the URLs below:
+         </p>
+         <ul>
+            <li>
+         <code>http://hostname:8080/jboss-as-logging-tools/rest/greetings/<b>name</b></code>
```
Comment (pmuir) [inline logging-tools/src/main/webapp/index.html]: Indenting appears to have gone weird here?

File: logging-tools/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+         <p>Simple use of localised messages (with parameter) and logging.</p>
+         <ul>
+            <li>Returns a localised "hello `name`" string where `name` is the last component of the URL.</li>
+            <li>Logs a localised "Hello message sent"</li>
+         </ul>
+               <p>Example: <a href="http://127.0.0.1:8080/jboss-as-logging-tools/rest/greetings/Harold">http://127.0.0.1:8080/jboss-as-logging-tools/rest/greetings/Harold</a></p>
```
Comment (pmuir) [inline logging-tools/src/main/webapp/index.html]: Should this be a relative URL to handle deployments to other than localhost and on other ports

**Comment by sgilda** [comment]
I just built and deployed this.

This URL still works: http://localhost:8080/jboss-as-logging-tools/rest/greetings/Harold, but I get a 404 when I access "http://localhost:8080/jboss-as-logging-tools/"

Isn't that the URL that should redirect to the index page? And should the README be updated with the this URL?

File: logging-tools/src/main/webapp/index.html (hunk 2 above)
Comment (dmison) [inline logging-tools/src/main/webapp/index.html]: Sent from my iPhone

On 31/07/2012, at 11:48 PM, Pete Muirreply@reply.github.com wrote:

> > -         courtesy of <a href="http://translate.google.com">http://translate.google.com</a> for demonstration.
> > -         My apologies if they are less than ideal translations.
> > -         </p>
> > -   
> > -         <p>
> > -         The different methods of the JAX-RS service can be access using the URLs below:
> > -         </p>
> > -         <ul>
> > -            <li>
> > -         <code>http://hostname:8080/jboss-as-logging-tools/rest/greetings/<b>name</b></code>
> > -         <p>Simple use of localised messages (with parameter) and logging.</p>
> > -         <ul>
> > -            <li>Returns a localised "hello `name`" string where `name` is the last component of the URL.</li>
> > -            <li>Logs a localised "Hello message sent"</li>
> > -         </ul>
> > -               <p>Example: <a href="http://127.0.0.1:8080/jboss-as-logging-tools/rest/greetings/Harold">http://127.0.0.1:8080/jboss-as-logging-tools/rest/greetings/Harold</a></p>
> 
> Should this be a relative URL to handle deployments to other than localhost and on other ports

_facepalm_ Doh! And other pop culture themed references to me missing the obvious. :-(

Fixing

**Comment by dmison** [comment]
- cleaned up indenting.  
- made the `<a href>` relative links, and moved them towards the top in each section.
- added reference to the `readme.md`

I did leave the non-linked ones because they indicate placeholders in the URLs.

**Comment by sgilda** [comment]
This works great now. I love the new landing page! 

I have one minor comment. In the heading, you use the British spelling for "Localisation" with an "s", yet everywhere else, you use the American spelling. Like I said, it's very minor.

Pete, do you have any more comments on this?

**Comment by pmuir** [comment]
I'm sorry, I missed your updates. Sande, please go ahead and merge :-)

**Comment by sgilda** [comment]
Merged.

**Pull request closed without merging**